### ISSUE TITLE
No solutions visible in Firefox Reader view

### ISSUE THREAD

**Issue by habere-et-dispertire** [exercism/exercism#6422]
Firefox has a reader view for removing some of the administrative detritus. Solutions are not visible in this mode.

**Comment by SleeplessByte** [comment]
@habere-et-dispertire assuming you use this to either "read" the page or have the page spoken to you, what would you like the solutions to read out? I'll make sure I'll help the team get the correct markup in place.

**Comment by SleeplessByte** [comment]
Understood. It will not expose all the iterations because they are actually not loaded (until you click the link). It not showing up at all is imo a bug, or at least something we should attempt to fix.

A print stylesheet is an option; I think we can even attempt to load a "print" variant of the syntax highlighting.

**Comment by habere-et-dispertire** [comment]
### Please also see [issue 520](https://github.com/exercism/raku/issues/520).

* * * *

Here's a monochrome highlight style which seems suitable for printing. It uses pandoc's automatic rendering from a fenced code section.

Notice also how using a typeface with understanding of common programming operators can harness ligatures to improve legibility. Point in case, the numeric equality operator `==` (two equals signs):

![is-achilles-min](https://user-images.githubusercontent.com/18032974/176068462-5f44b48a-3db9-4982-ac83-52f97ee0e498.png)

**Comment by SleeplessByte** [comment]
Yes, ligatures are great for this.

**Comment by habere-et-dispertire** [comment]
Just to over communicate and clarify, I think that a print stylesheet would be well suited to not only printouts, but to viewing *solutions* on screen. Colour syntax highlighting is geared towards *editing*.

**Comment by SleeplessByte** [comment]
I think that's very much a "personal preference". In my opinion, the better solution would to be able to select in what cases you (as a user) would see which type of styling.

**Comment by habere-et-dispertire** [comment]
Yes and for some reason it seems to get heated fast (see the comments section under Linus Åkesson's article: [ A case against syntax highlighting](https://linusakesson.net/programming/syntaxhighlighting/).

So a personal preference here sounds sound. A will not argue beyond for a default against prevailing views to the contrary.

**Comment by SleeplessByte** [comment]
Oh yeah. I don't see the point in replicating that here. You pointed out that it matters to you and the impact on the ease for you to consume the content. I think that, up to a reasonable point, should be enough to consider this change 🤗.

I'll let you know if I have more questions about this but i think your examples are very clear.
